packages/api-generator/src/index.js

```diff
@@ -262,3 +272,29 @@ components['$vuetify'] = map['$vuetify']
 components['internationalization'] = map['internationalization']
 
 writeApiFile({ ...components, ...directives }, 'dist/api.js')
+
+// Create web-types.json to provide autocomplete in JetBrains IDEs
+const webTypes = {
+  // $schema: "../../schema/web-types.schema.json",
+  framework: 'vue',
+  name: 'vuetify',
+  version: pkg.version,
+  contributions: {
+    html: {
+      'types-syntax': 'typescript',
+      tags: [],
+    },
+  },
+}
+
+const webTypesTags = { ...components, ...directives }
+Object.keys(webTypesTags).forEach(function (key) {
+  const name = capitalize(camelize(key))
```

Comment:
can be done with one single method if I introduce a new method (say `pCamelize`) for pascal case.